finagle-memcached/src/main/scala/com/twitter/finagle/memcached/Client.scala

```diff
@@ -637,12 +638,12 @@ protected class ConnectedClient(protected val service: Service[Command, Response
   import ClientConstants._
 
   protected def rawGet(command: RetrievalCommand): Future[GetResult] = {
-    val keys: immutable.Set[String] = command.keys.map { case Buf.Utf8(s) => s }(breakOut)
+    val keys: immutable.Set[String] = command.keys.iterator.map { case Buf.Utf8(s) => s }.toSet
 
-    service(command).transform {
+    service(command).transform{ case x => {println(s"transformed $x"); x match {
```

Comment:
assuming this is for debugging, we can remove it when pulling in.

Reply:
How did I miss that...

Yes, that needs to be removed